src/node_api.cc

```diff
@@ -3088,6 +3088,18 @@ napi_status napi_get_version(napi_env env, uint32_t* result) {
   return napi_clear_last_error(env);
 }
 
+napi_status napi_get_node_version(napi_env env,
+                                  uint32_t* versions,
+                                  const char** release) {
```

Comment:
Having a string out parameter is a little bit risky since there needs to be a clear chain of ownership.  In this case it will always be a constant string, but as a developer it may not be clear how long I can hold the string for and whether I need to free the string myself.